ratelimit_test.go

```diff
@@ -102,6 +115,7 @@ func (r *runnerImpl) startTaking(rls ...Limiter) {
 			for _, rl := range rls {
 				rl.Take()
 			}
+			r.clock.advance(time.Nanosecond, 0)
```

Comment:
and we need this one as well?

Reply:
Yes, this one imitates time progressions between `rl.Take()` calls. With a real clock, when you call `rl.Take()` do something and call `rl.Take()` again `time.Now()` will move forward in time, so we want to imitate that.